homeassistant/components/kostal_plenticore/const.py

```diff
@@ -618,11 +618,25 @@
         {ATTR_UNIT_OF_MEASUREMENT: PERCENTAGE, ATTR_ICON: "mdi:battery-negative"},
         "format_round",
     ),
+]
+SWITCH_SETTINGS_DATA = [
```

Comment:
This is really quite unreadable, and the explanatory comment for `SENSOR_SETTINGS_DATA` doesn't seem to match `SWITCH_SETTINGS_DATA` and `SELECT_SETTINGS_DATA`.

Make these named tuples, dataclasses or maybe typed dicts to make them readable;

```
@dataclass
class SwitchSettings:
    ...

SWITCH_SETTINGS_DATA: list[SwitchSettings] = [
    ...
]
```

Reply:
Tried to solve that.

Reply:
Nice!